src/test/java/sandbox/InfluxdbUdpSandbox.java

```diff
@@ -18,6 +18,7 @@ public static void main(String[] args) {
 		try {
 			final MetricRegistry registry = new MetricRegistry();
 			reporter = getInfluxdbReporter(registry);
+//			reporter = getInfluxdbV9Reporter(registry);
```

Comment:
Please do not add disabled code. A possible alternative is to pass in the version as a command line argument.